guides/common/modules/proc_enrolling-project-server-in-freeipa-domain.adoc

```diff
@@ -0,0 +1,116 @@
+endif::[]
+. On {ProjectServer}:
+.. Configure the {ProjectServer} system as client in the {FreeIPA} domain.
+This includes ensuring that the system meets the necessary prerequisites, installing the necessary packages, and running the {FreeIPA} client installation utility.
++
+[NOTE]
+====
+To install the {FreeIPA} client package, use the `{foreman-installer}` utility.
+====
+ifdef::satellite[]
++
+For more information, see link:{RHELDocsBaseURL}/8/html-single/installing_identity_management/index#assembly_installing-an-idm-client_installing-identity-management[{RHEL}{nbsp}8 Installing Identity Management] or link:{RHELDocsBaseURL}/9/html-single/installing_identity_management/index#assembly_installing-an-idm-client_installing-identity-management[{RHEL}{nbsp}9 Installing Identity Management].
+endif::[]
+ifdef::foreman-deb[]
++
+. Ensure that the hostname is set to the fully qualified domain name (FQDN); the short name is not sufficient:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# hostname
+{foreman-example-com}
+----
++
+Otherwise, `{foreman-installer}` cannot generate the right principal name that is needed to join the realm.
+endif::[]
+
+.Verification
+
+* On your {ProjectServer}, check that you are able to resolve a user defined on the {FreeIPA} server.
+For example, to check the `admin` user that {FreeIPA} creates by default:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ id admin
+----
+
+.Enrolling a {ProjectServer} system as a {FreeIPA} client from the command line by using a one-time password enrollment
+====
+On the {FreeIPA} server, a user named _admin_ who has administrative privileges on the {FreeIPA} server prepares a host entry for the {ProjectServer} system:
+
+. Authenticate as the {FreeIPA} _admin_ user:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# kinit _admin_
+----
+. Optional: Verify that you have authenticated successfully:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# klist
+----
+. Create a host entry from the command line.
+Specify that you want to use a random password for the enrollment.
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# ipa host-add --random _{project-context}-server.example.com_
+--------------------------------------------------
+ Added host "{project-context}-server.example.com"
+ --------------------------------------------------
+  Host name: {project-context}-server.example.com
+  Random password: W5YpARl=7M.n
+  Password: True
+  Keytab: False
+  Managed by: ipa-server.example.com
+----
+. Create an HTTP service entry for your {ProjectServer}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# ipa service-add HTTP/_{project-context}-server.example.com_
```

Comment:
TO DO: When we progress with testing we should verify that "_{project-context}-server.example.com_" is complete. It might be "_{project-context}-server.example.com_@_<realm_name>_"

Reply:
No change necessary. It worked for me on two different IPA servers:
```
ipa service-add HTTP/foreman.example.com
------------------------------------------------------------
Added service "HTTP/foreman.example.com@EXAMPLE.COM"
------------------------------------------------------------
  Principal name: HTTP/foreman.example.com@EXAMPLE.COM
  Principal alias: HTTP/foreman.example.com@EXAMPLE.COM
  Managed by: foreman.example.com
```
I don't know why the command failed for us before but let's blame DNS as usual and move on :)